corehq/apps/export/views.py

```diff
@@ -1831,11 +1820,10 @@ def get_app_data_drilldown_values(self, in_data):
         try:
             rmi_helper = ApplicationDataRMIHelper(self.domain, self.request.couch_user)
             response = rmi_helper.get_case_rmi_response()
-        except Exception as e:
+        except Exception:
             return format_angular_error(
-                _("Problem getting Create Export Form: {}").format(e.message),
+                _("Problem getting Create Export Form"),
                 log_error=True,
-                exception=e,
```

Comment:
Heads up, what I said about `logging.exception` wasn't quite right—it doesn't log to celery. (The other stuff was true.) I changed this in https://github.com/dimagi/commcare-hq/pull/16993 to use `notify_exception`, which will go to sentry. I could have used logging.getLogger('notify').exception instead with similar effect, but that didn't seem to be used anywhere, so I went with our internal convention and used `notify_exception`.